webapp/components/Logo/Logo.vue

```diff
@@ -53,7 +53,7 @@ export default {
 .ds-logo-svg {
   width: 130px;
   height: auto;
-  fill: currentColor;
+  fill: #000000;
```

Comment:
This will need to be changeable eventually - but I guess we can do that with a refactoring that actually requires a different color schema

Reply:
Usually it makes no sense here to fill black areas of a normally fixed coloured logo with the current color. In this case all black areas of our ocelot.social logo got green.
Therefore the standard (current) fill color should be black. @ulfgebhardt 

If I remove the line it gets green as well …
But I admit that I haven't investigated this further from where this behaviour comes..